src/OpenSpaceToolkit/Astrodynamics/EventCondition/AngularCondition.cpp

```diff
@@ -36,19 +47,14 @@ AngularCondition::Criterion AngularCondition::getCriterion() const
     return criterion_;
 }
 
-std::function<Real(const State&)> AngularCondition::getEvaluator() const
-{
-    return evaluator_;
-}
-
 Angle AngularCondition::getTargetAngle() const
 {
-    if (!target_.isDefined())
+    if (!target_.value_.isDefined())
```

Comment:
nit: if `value_` has an underscore, I would usually expect it to be private to the Target class and not accessed from outside sources. Worth making a `getValue()` function?

Reply:
Excellent point.